packages/amplify-category-function/provider-utils/awscloudformation/cloudformation-templates/lambda-cloudformation-template.json.ejs

```diff
@@ -121,9 +121,27 @@
                             "Action": ["dynamodb:GetItem","dynamodb:Query","dynamodb:Scan","dynamodb:PutItem","dynamodb:UpdateItem","dynamodb:DeleteItem"],
                             "Resource": [
                             <% if (props.database && props.database.Arn) { %>
-                                "<%= props.database.Arn %>"
+                                "<%= props.database.Arn %>",
+                                {
+                                    "Fn::Join": [
+                                        "/",
+                                        [
+                                            "<%= props.database.Arn %>",
+                                            "index/*"
```

Comment:
"index/*" gives access to GSI's as well as LSI's?

Reply:
The Dynamo DB does not have separate access permission for GSI's and LSI's. 

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/api-permissions-reference.html